File: Python/program23.py
def main():
    
    iValue = int(input("Enter number \n"))
    
    bRet = Check_Divisible(iValue)
    
    if(bRet == True):
        print("Number is divisble by 5 ")
    else:
        print("Number is not divisble by 5 ")
    
def Check_Divisible(iNo):
    bflag = False
    if((iNo % 5) == 0):
        bflag = True
    else:
        bflag = False
    return bflag
    

    
    
if __name__ == "__main__":
    main()

File: Java/program23.java
import java.util.*;

class Divisible
{
    public boolean CheckDivisible(int iNo)
    {
        boolean bResult = false;
        if((iNo % 5) == 0 && (iNo % 3) == 0)
        {
            bResult = true;
        }
        else
        {
           bResult = false;
        }
        
        return bResult;
    } 
}

class program23
{
    public static void main(String A[])
    {
        int iValue = 0;
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number :");
        iValue = sobj.nextInt();

        Divisible dobj = new Divisible();
        bRet = dobj.CheckDivisible(iValue);

        if(bRet == true)
        {
            System.out.println(iValue +" is divisible by 3 and 5");

        }
        else
        {
            System.out.println(iValue +" is not divisible  by 3 and  5");
        }

               

        
    }
}